src/components/table/Table.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react'
+import './Table.scss'
+
+const Table = ({ tblData = [], setEditModal, setDeleteModal }) => {
+  const [property, setProperty] = useState([])
+  useEffect(() => {
+    if (tblData.length) {
+      setProperty(Object.keys(tblData[0]))
+    }
+  }, [])
+  return (
+    <div className='tbl-wrapper'>
+      <table className='tbl'>
+        <thead className='tbl__heading'>
+          <tr>
+            <th>S.N.</th>
```

Comment:
columns for table should be passed as some table might not have SN or option or might have to just show few columns not all columns of record.
In category case updated and created date are not required